integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/kotlin/StdlibGradleIntegrationTest.kt

```diff
@@ -23,18 +23,20 @@ class StdlibGradleIntegrationTest(override val versions: BuildVersions) : Abstra
 
     override val projectOutputLocation: File by lazy { File(projectDir, "build/dokka/kotlin-stdlib") }
 
+    private val currentDokkaVersion: String = "1.5.0"//checkNotNull(System.getenv("DOKKA_VERSION")) // uncomment after updating of StdLib
```

Comment:
Am I correct in understanding that stdlib integration test will be running Dokka 1.5 until we update? If so, it's a bit worrying since we use this test to showcase features and, well, to test Dokka :)

Do you have any plans of updating these tests, given that you said it's not very easy? It's one thing if it's updated in a week or two, and another if it's not planned and we'll have broken stdlib tests for months

Reply:
> Am I correct in understanding that stdlib integration test will be running Dokka 1.5 until we update?

No,  it will be still running `dokka-gradle-plugin:for-integration-tests-SNAPSHOT` with actual Dokka.

`currentDokkaVersion` is only used here to build plugins (as `compileOnly`).

> Do you have any plans of updating these tests, given that you said it's not very easy?

No, now we can not use the plugin to ignore common builtIns due to it should depend on the last Dokka rather than 1.5.